ShareClientUI/ShareClientSettingsViewController.swift

```diff
@@ -15,16 +16,30 @@ public class ShareClientSettingsViewController: UITableViewController {
 
     public let cgmManager: ShareClientManager
 
-    private var glucoseUnit: HKUnit
+    private let displayGlucoseUnitObservable: DisplayGlucoseUnitObservable
+
+    private var cancellable: AnyCancellable?
```

Comment:
[nit] I prefer using `private lazy var cancellables = Set<AnyCancellable>()` and `.store(in: &cancellables)`, but that's just me.